### ISSUE TITLE
PuLID stays persistent

### ISSUE THREAD

**Issue by MoonMoon82** [sipie800/ComfyUI-PuLID-Flux-Enhanced#21]
It seems, once a sampling process with Flux PuLID started, it stays persistent even if the node is removed:

1st samling process:
![grafik](https://github.com/user-attachments/assets/7a968b76-cb11-43fc-b4dc-2a67129bb9e4)
2nd sampling process:
![grafik](https://github.com/user-attachments/assets/b1ef0bd6-a56d-437e-be99-91f6bd8dfa81)

Could you please check?

Kind regards!

**Comment by sipie800** [comment]
perhaps you loose the seed and run again

**Comment by MoonMoon82** [comment]
@sipie800 the PuLID still stays persistent even I change the seed:
![image](https://github.com/user-attachments/assets/65bfe3ad-5aca-475a-98bf-6d282e789b5f)

**Comment by jaszczw** [comment]
My workaround is to set weight to 0, which 'resets' the model - not perfect but works for my use case.

https://github.com/balazik/ComfyUI-PuLID-Flux/issues/41 - this is the original issue

**Comment by MoonMoon82** [comment]
@jaszczw Unfortunately that does not appear to be a proper workaround. I found this issue while I was trying to integrate FluxPuLID into a more complex workflow which samples at first without FluxPuLID, later with FluxPuLID. As soon I restart the whole workflow all sample processes are using FluxPuLID.

**Comment by jaszczw** [comment]
Yeah, you would need to put pulid with basically anything in it with weight 0 - which I do.

**Comment by MoonMoon82** [comment]
@jaszczw Your suggestion does not work:
![image](https://github.com/user-attachments/assets/2c829f0e-5f73-45c1-951f-5435c1cf2c04)

**Comment by cchance27** [comment]
This is because Apply pulid flux is monkeypatching the unet, the current comfyui modelpatcher doesn't support the required block patching needed for plux so it's basically a hack (all implementations for flux not just this repo). 

So if you use apply pulid, that unet is officially THAT pulid, it can't just be swapped again later or in a different line like above... 

Use a different unet loader into the second apply pulid, though i dont know if that will work given internal comfy caching of models, but worth a try.

**Comment by vararth** [comment]
Is there no way to clean the unet after the workflow? Just remove the embeddings PuLID adds?
